src/ColorContextMenu-Core.package/ColorChooserMorph.class/instance/buildDiscreteMenu.attribute.levels..st

```diff
@@ -1,12 +1,18 @@
 menus
 buildDiscreteMenu: menuFactory attribute: aSymbol levels: levels
 
-	| menu setter |
+	| menu setter getter steps item |
+	self flag: 'Warum heißt es hier menuFactory und nicht aRadialMenuMorph?'.
 	menu := menuFactory value.
-	setter := ('set{1}:'  format: {aSymbol capitalized}) asSymbol.
+	setter := ('set{1}:' format: {aSymbol capitalized}) asSymbol.
 	
-	0 to: levels - 1 do: [:level |
-		menu add: ('{1}%' format: {(level / (levels - 1) * 100) rounded}) action: [
-			self perform: setter with: level / (levels - 1)]].
+	steps := levels - 1.
+	0 to: steps do: [:level |
+		item := menu 
+			add: ('{1}%' format: {(level / steps * 100) rounded}) 
+			action: [self perform: setter with: level / steps].
+		item class = RadialMenuItemMorph ifTrue: [
+			getter := ('get{1}:' format: {aSymbol capitalized}) asSymbol.
+			item baseColor: (self perform: getter with: level / steps)]].
```

Comment:
Let's rework this to respect the `#colorizeMenuItem:color:` change from https://github.com/LinqLover/ColorContextMenu/pull/17/commits/5cd15b23516e35a6306c6141afb2997dd4ef4bb3 when it's ready :)

Apart from that, instead of `item class = RadialMenuItemMorph` always write `(item isKindOf: RadialMenuItemMorph)` :) We should refactor this anyway later ...

Reply:
done the second part (isKindOf:)

Reply:
moved to new issue